api/src/test/java/io/minio/MinioClientTest.java

```diff
@@ -30,11 +30,10 @@
 import java.security.NoSuchAlgorithmException;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Calendar;
-import java.util.Iterator;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
Its required to import specifically than all. It is done by some code analysis tools guidelines.